core/templates/services/state-interaction-stats.service.ts

```diff
@@ -99,7 +90,11 @@ export class StateInteractionStatsService {
    * Returns a promise which will provide details of the given state's
    * answer-statistics.
    */
-  computeStats(state: State): Promise<IStateRulesStats> {
+  computeStats(
+      state: State, useCache: boolean = false): Promise<IStateRulesStats> {
```

Comment:
Pass in the arg explicitly each time, we try to avoid default args.

Reply:
Done.

Reply:
Not done?

Reply:
Hadn't pushed yet, done

Reply:
Ah, one note -- in the future, please use "Start a review", make the code changes while addressing comments, push the code changes, and _only then_ submit your review.

Once the comment appears on github, the assumption is that the code changes have been done.

Reply:
Ack!